operator/src/main/resources/scripts/startServer.sh

```diff
@@ -215,13 +215,18 @@ function prepareMIIServer() {
     trace  SEVERE "Domain Source Type is FromModel, cannot create ${DOMAIN_HOME}/lib "
     return 1
   fi
+  local WLSDEPLOY_DOMAINLIB="wlsdeploy/domainLibraries"
 
   for file in $(sort_files ${IMG_ARCHIVES_ROOTDIR} "*.zip")
     do
         # expand the archive domain libraries to the domain lib
         cd ${DOMAIN_HOME}/lib || return 1
-        ${JAVA_HOME}/bin/jar xf ${IMG_ARCHIVES_ROOTDIR}/${file} wlsdeploy/domainLibraries/
-
+        ${JAVA_HOME}/bin/jar xf ${IMG_ARCHIVES_ROOTDIR}/${file} ${WLSDEPLOY_DOMAINLIB}
+        # Flatten the jars to the domain lib root
+        if [ -d ${WLSDEPLOY_DOMAINLIB} ] && [ "$(ls -A ${WLSDEPLOY_DOMAINLIB})" ] ; then
+          mv ${WLSDEPLOY_DOMAINLIB}/* .
+          rm -fr wlsdeploy/
```

Comment:
should this call rm -fr regardless of whether the domain lib was found in the wlsdeploy directory?